CHANGELOG.md

```diff
@@ -41,10 +41,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 - (erc20) [#2154](https://github.com/evmos/evmos/pull/2154) Remove EVM hooks from `erc20` module.
 - (erc20) [#2155](https://github.com/evmos/evmos/pull/2155) Remove CLI commands for register and convert Coin.
+- (erc20) [#2159](https://github.com/evmos/evmos/pull/2159) Update IBC middleware with automatic coin conversion of the single token representation v2.
```

Comment:
I see there're many changes on the Changelog.
Is this intentional?

Reply:
Yes, this is also fixing a bunch of changelog entries that have been added to the `v16.0.0` tag after it was released 👍